src/libraries/Microsoft.Extensions.Logging.Configuration/src/LoggerProviderConfigurationExtensions.cs

```diff
@@ -12,13 +13,16 @@ namespace Microsoft.Extensions.Logging.Configuration
     /// </summary>
     public static class LoggerProviderOptions
     {
+        internal const string TrimmingRequiresUnreferencedCodeMessage = "Some parts of TOptions and its dependent types may be trimmed. Ensure the necessary constructors and properties are preserved.";
```

Comment:
I think the `src` file needs updating to match the `ref`.

We should also update the other libraries' messages as well to read the same.

Reply:
good catch, will fix.